configuration.yml

```diff
@@ -7,7 +7,7 @@ dependencies:
   - notebook>=5.7.6
   - tornado>6
   - ipywidgets>=7.0.0
-  - nodejs
+  - nodejs=10.13.0
```

Comment:
@LeeTZ had to do this because of #8028 (on clean env that I had its a must)

Reply:
Fixing isn't a long term solution, we should revisit this later